install/install-oasis

```diff
@@ -49,22 +50,32 @@ cp $SCRIPTDIR/aufs.postupdate /opt/etc/osupdate.d
 /opt/etc/osupdate.d/aufs.postupdate
 
 # create srv directory if it isn't there yet
-SRVDIR="/net/nas01/srv/`uname -n|cut -d. -f1`"
-if [ ! -d $SRVDIR ]; then
-    mkdir $SRVDIR
-    setfacl -b $SRVDIR
-    chmod 755 $SRVDIR
-    mkdir $SRVDIR/cvmfs
-    mkdir $SRVDIR/spool
+NETSRVDIR="/net/vm08/oasis-share/srv/$MYSHORTNAME"
+if [ ! -d $NETSRVDIR ]; then
+    mkdir $NETSRVDIR
```

Comment:
I had thought about that but there's never actually a chance for it to make a difference because later on the itb expects a file to be there in non-itb, and we never do production before itb.  Also the directory is known to not be there because of the ! -d.